.gitignore

```diff
@@ -1,3 +1,6 @@
+# Project-specific files
+sampleproject/staticfiles/
```

Comment:
`sampleproject/staticfiles/` is already in .gitignore of nested `sampleproject/.gitignore`. But for simplicity the script doesn't handle nested `.gitignores`, so added this pattern here.